hack/post-run-e2e.sh

```diff
@@ -20,8 +20,10 @@ export KUBECONFIG="${MAIN_KUBECONFIG}"
 kubectl --context="${HOST_CLUSTER_NAME}" delete -f "${REPO_ROOT}"/examples/customresourceinterpreter/karmada-interpreter-webhook-example.yaml
 
 # uninstall metallb
-kubectl --context="${HOST_CLUSTER_NAME}" delete configmap config -n metallb-system
-kubectl --context="${HOST_CLUSTER_NAME}" delete -f https://raw.githubusercontent.com/metallb/metallb/v0.13.5/config/manifests/metallb-native.yaml
+curl https://raw.githubusercontent.com/metallb/metallb/v0.13.5/config/manifests/metallb-native.yaml -k | \
+  sed '0,/args:/s//args:\n        - --webhook-mode=disabled/' | \
+  sed '/apiVersion: admissionregistration/,$d' | \
+  kubectl --context="${HOST_CLUSTER_NAME}" delete -f -
```

Comment:
Why shall we sed it when deleting?

Reply:
command `sed '/apiVersion: admissionregistration/,$d'` will delete one `ValidatingWebhookConfiguration`, if not delete it before, `kubectl apply` will have erros:
```shell
jw@ecs-3fa1 [04:23:37 PM] [~]
-> % kubectl delete -f https://raw.githubusercontent.com/metallb/metallb/v0.13.5/config/manifests/metallb-native.yaml
namespace "metallb-system" deleted
customresourcedefinition.apiextensions.k8s.io "addresspools.metallb.io" deleted
customresourcedefinition.apiextensions.k8s.io "bfdprofiles.metallb.io" deleted
customresourcedefinition.apiextensions.k8s.io "bgpadvertisements.metallb.io" deleted
customresourcedefinition.apiextensions.k8s.io "bgppeers.metallb.io" deleted
customresourcedefinition.apiextensions.k8s.io "communities.metallb.io" deleted
customresourcedefinition.apiextensions.k8s.io "ipaddresspools.metallb.io" deleted
customresourcedefinition.apiextensions.k8s.io "l2advertisements.metallb.io" deleted
serviceaccount "controller" deleted
serviceaccount "speaker" deleted
role.rbac.authorization.k8s.io "controller" deleted
role.rbac.authorization.k8s.io "pod-lister" deleted
clusterrole.rbac.authorization.k8s.io "metallb-system:controller" deleted
clusterrole.rbac.authorization.k8s.io "metallb-system:speaker" deleted
rolebinding.rbac.authorization.k8s.io "controller" deleted
rolebinding.rbac.authorization.k8s.io "pod-lister" deleted
clusterrolebinding.rbac.authorization.k8s.io "metallb-system:controller" deleted
clusterrolebinding.rbac.authorization.k8s.io "metallb-system:speaker" deleted
secret "webhook-server-cert" deleted
service "webhook-service" deleted
deployment.apps "controller" deleted
daemonset.apps "speaker" deleted
Error from server (NotFound): error when deleting "https://raw.githubusercontent.com/metallb/metallb/v0.13.5/config/manifests/metallb-native.yaml": validatingwebhookconfigurations.admissionregistration.k8s.io "metallb-webhook-configuration" not found

```

Reply:
@ikaven1024 Do you want to say, why not use `--ignore-not-found`?

```
$ kubectl delete --help
    --ignore-not-found=false:
	Treat "resource not found" as a successful delete. Defaults to "true" when --all is
	specified.

```

Reply:
fixed